src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -150,9 +150,6 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
         }
     }
 
-    /**
-     * {@inheritdoc}
-     */
```

Comment:
These changes must not belong to that patch. You should ignore theses fabbot errors for that PR as it doesn't belong 
 to your work :slightly_smiling_face:

Reply:
Alright. I'll drop the code changes commit then.